.github/workflows/lint-and-test.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint and Test Charts
+
+on: pull_request
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Why are we fetching everything here?  Is that necessary for these jobs?

Reply:
tbh, I copied it from the example, but iirc it was necessary for `ct lint` and related commands to figure out which charts had changed; by default, `actions/checkout` does a shallow checkout.

Reply:
Yeah, that makes sense for identifying what changed.  I wonder if it could do `fetch-depth: 2` or something so its not getting everything.   Probably an early optimization for this repo but if we moved this CI/CD into core custodian it'd hurt to pull everything

Reply:
The issue with setting it to a fixed number is that the number has to be at least as large as the number of commits to be merged, which is ... fuzzy. To be safe, maybe something like 20 or 50 would make sense, but at that point .... :shrug: 

That said, I think there's value in keeping the helm charts separate. They have a lot of opinions about tags that should be made, release assets, gh-pages for the chart repo, etc, that would pollute the main repo.